Automattic/wp-calypso
# Plans Redesign: conditionally display plan features

Fix for #6500. With redesigned plans, we show all plans features. However, some features are behind abtests. This PR changes `plansList` according to active abtests.

## Testing Instructions

These are the active abtests that touch the `/plans` page and/or `/start/plans` page in some way:

- `wordadsInstantActivation`;
- `googleVouchers`;
- `wordpressAdCredits`.

While following the testing instructions below, enable/disable the abtests above and check if it changes the behavior of the `/plans` and `/start/plans` pages.

1. Run Calypso with `ENABLE_FEATURES=manage/plan-features make run`;
2. Disable all abtests;
3. Navigate to `/plans` page and verify that the features/descriptions from the abtests are not present;
4. Enable abtests one by one and check whether it changes the aforementioned pages.

Note: `wordadsInstantActivation` is visible only on the `/start/plans` page because it is enabled only for new users. Or if you have a brand new account, you could (probably) see it on `/plans` page.

To the authors of the aforementioned abtests: could you please double-check everything works as it should? Thanks!

## Screenshots

With all the aforementioned abtests enabled:

![selection_028](https://cloud.githubusercontent.com/assets/4988512/16626983/84b1cb0e-43ab-11e6-9ae4-3493ffe968ac.png)

With only `googleVouchers` abtest enabled:

![selection_029](https://cloud.githubusercontent.com/assets/4988512/16626989/8ca42dde-43ab-11e6-90c3-5e783406de37.png)



**Note:** I based this PR on #6376 so please ignore the commits before `Add fn to check for wordAdsInstantActivation`.

cc @gwwar @rralian @artpi @retrofox @mtias 


Test live: https://calypso.live/?branch=add/plans-redesign-conditionally-display-plan-features
